fv3core/stencils/d_sw.py

```diff
@@ -1010,5 +1033,12 @@ def __call__(
             heat_source,
             diss_est,
             self._column_namelist["d_con"],
+        )
+
+        self._update_u_and_v_stencil(
```

Comment:
I think my strategy for merging this with my own d_sw changes will be to keep my changes, and then use your changes as a guide to fix the gtc tests I'll break by not having taken your changes. It's pretty easy to understand what these changes are thanks to your comments.